_automating-configurations/api/create-workflow.md

```diff
@@ -89,7 +100,7 @@ The following table lists the available request fields.
 |:---	|:---	|:---	|:---	|
 |`name`	|String	|Required	|The name of the workflow.	|
 |`description`	|String	|Optional	|A description of the workflow.	|
-|`use_case`	|String	|Optional	| A use case, which can be used with the Search Workflow API to find related workflows. In the future, OpenSearch may provide some standard use cases to ease categorization, but currently you can use this field to specify custom values.	|
+|`use_case`	|String	|Optional	| A user-provided use case, which can be used with the Search Workflow API to find related workflows. You can use this field to specify custom values. This is distinct from the `use_case` query parameter. |
```

Comment:
I wish this parameter was named differently, otherwise it may be confusing. Thank you for adding the clarification here.

Reply:
Yeah, I agree. I think our vision for pre-provided templates changed somewhat from our initial template, where `use_case` is somewhat unused and should probably have been `tag`.